tools/wake/describe.cpp

```diff
@@ -312,6 +312,42 @@ void describe_timeline(const std::vector<JobReflection> &jobs,
          "</html>\n";
 }
 
+void describe_simple_timeline(const std::vector<JobReflection> &jobs,
```

Comment:
should probably make a shared function to DRY it up here but each rendering feels subtly different in such a way that following a DRY path here feels incorrect.